### DISCUSSION TITLE
How to annotate dataset for finetune..

### DISCUSSION THREAD

**Discussion by tzktok** [PaddlePaddle/PaddleOCR#13189] [Q&A]
#### 问题描述 / Problem Description

https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/finetune_en.md
 
how to annotate dataset for en_PP-OCRv3_det model ? @GreatV 
pls provide any labeled txt file..

**Comment by GreatV** [comment]
hi @tzktok, please refer to: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/dataset/ocr_datasets_en.md#11-paddleocr-text-detection-format-annotation
